pkg/kubecfg/githubTeams_test.go

```diff
@@ -0,0 +1,29 @@
+package kubecfg
+
+import (
+	"reflect"
+	"testing"
+)
+
+func eq(t *testing.T, got interface{}, expected interface{}) {
+	t.Helper()
+	if !reflect.DeepEqual(got, expected) {
+		t.Errorf("expected: %v, got: %v", expected, got)
+	}
+}
+
+func TestGetTokenClaims(t *testing.T) {
+	token := "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkFsZWphbmRybyBHYXJyaWRvIE1vdGEiLCJlbWFpbCI6ImFsZWphbmRyby5nYXJyaWRvQGRpZ2l0YWwuanVzdGljZS5nb3YudWsiLCJodHRwczovL2s4cy5pbnRlZ3JhdGlvbi5kc2QuaW8vZ3JvdXBzIjpbbnVsbCwiZ2l0aHViOndlYm9wcyIsbnVsbF0sIm5pY2tuYW1lIjoibW9nYWFsIiwiaWF0IjoxNTE2MjM5MDIyfQ._5Vkfh1lDWQvV9xIcbpnMFWOnkJBzCcp1X7lYcEG9sE"
```

Comment:
As per my question above, these are real-looking but not actual secrets, right?

Reply:
Yes, this token was generated used https://jwt.io/ , you can write fields you want in the "Decoded" side and it builds the token for you.